org.openhab.binding.zigbee/README.md

```diff
@@ -153,23 +153,23 @@ The following coordinators are known to be supported.
+| [Texas Instruments CC2531EMK](http://www.ti.com/tool/cc2531emk)                                                                                       | [CC2531](#cc2531-coordinator)        |                                                          | CC2531 is not recommended for more then 15 devices due to its less powerful MCU. Also needs extra [hardware and Z-Stack Home 1.2 firmware flashed](https://www.zigbee2mqtt.io/guide/adapters/flashing/flashing_the_cc2531.html). CC2530 and CC2538 may work too with Z-Stack Home 1.2 compatible firmware. Not recommended today because its old hardware is no longer getting firmware updates. |
 
 #### Ember EZSP NCP Coordinator
 
-The Ember EZSP NCP (Network Co-Processor) supports the SiLabs MightyGecko (EFR32MG) and the older EM357/8 dongles with the standard NCP firmware. The thing type is `coordinator_ember`. This dongle type is recommended due to its extensive use within both the community, and commercial systems using this dongle.
+The Ember EZSP NCP (Network Co-Processor) supports Silicon Labs Mighty Gecko (EFR32MG Family) and the older EM357/8 dongles with Silabs standard EmberZNet NCP Zigbee Coordinator firmware. The thing type is `coordinator_ember`. Ember dongles are recommended due to its extensive use within both the community and commercial systems using EmberZNet based hardware. Recommendation today is to get EFR32MG21 (EFR32 Mighty Gecko Series 2) or EFR32MG12 (EFR32 Mighty Gecko Series 1) based dongle as they support the latest EmberZNet NCP firmware.
 
 Note that there are generally two versions of the Ember NCP firmware in use. One operates at a baud rate of 115200 with RTS/CTS flow control (i.e. hardware flow control), the other operates at a baud rate of 57600, and XON/XOFF flow control (i.e. software flow control). If you are programming your own stick (e.g. the CEL stick) then it should be advisable to use the hardware flow control version - many commercial sticks seem to use the lower speed and software flow control (e.g. Bitron and Nortek HUSBZB-1).
 
-Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information.
+Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information. Short summery of what versions is compatible with which hardware chips is that newer dongles are not compatible with older firmware just as older dongles usually not compatible with newer firmware.
```

Comment:
I'd suggest to remove this - for starters this makes no sense, but it's also not really relevant. For the most part it makes no difference from a firmware perspective - the only real difference relates to chipsets with 256k of memory or less which cannot run older firmware.

Reply:
Changed it moving the small part I still think is relative from and removing the rest.